spec/controllers/hub/clients_controller_spec.rb

```diff
@@ -511,11 +511,13 @@
         params[:client][:action] = "set"
       end
 
-      it "adds attention_needed_since to client" do
+      it "adds attention_needed_since to client and touches last_incoming_interaction_at" do
         client.clear_attention_needed
-        patch :attention_needed, params: params
-        client.reload
-        expect(client.attention_needed_since).to be_present
+        expect {
+          patch :attention_needed, params: params
+          client.reload
+        }.to change(client, :attention_needed_since)
+         .and change(client, :last_incoming_interaction_at)
       end
     end
```

Comment:
I think it would make sense to have two contexts for setting needs attention: the context where a client does not need attention (as already described here) and another where the client already needs attention (to make sure we don't inadvertently reduce the duration of attention_needed_since)